README.md

```diff
@@ -1 +1,115 @@
-# lab-16
\ No newline at end of file
+# lab-16
+
+
+
+##  Event Driven Application
+
+### Author: Blandine Dasilveira
+
+### Links and Resources
+
+
+- pickup:  new order needs to be delivered
+
+- in-transit: order  in the process of being delivered
+
+- delivered: order has been delivered
+
+- Application should generate random orders every 5 seconds, with store, id, customer, and address as the order data.

```

Comment:
You can put this under [How to use your library](https://github.com/codefellows/seattle-javascript-401n16/blob/master/reference/submission-instructions/labs/README-template.md) as referred in the template.